clients/gtest/gemv_gtest.yaml

```diff
@@ -52,7 +52,7 @@ Tests:
   category: pre_checkin
   function: gemv
   precision: *single_double_precisions
-  transA: [ N, T, C ]
+  transA: [ N, T ]
```

Comment:
We need to test `C` because it's a valid trans value, even for reals.

Reply:
Right! Adding complex conjugate to all the tests